checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
```

Comment:
We should avoid needing this in individual checks. This should be pulled into parent class if need be.